spark/src/test/scala/geotrellis/spark/io/PersistenceSpec.scala

```diff
@@ -47,6 +49,16 @@ abstract class PersistenceSpec[K: ClassTag, V: ClassTag] extends FunSpec with Ma
       actual should contain theSameElementsAs expected
     }
 
+    it("should update a layer") {
+      updater.update(layerId, sample)
+    }
+
+    it("should not update a layer") {
```

Comment:
Can you make this test name more descriptive? What's the reason this update should fail?